viewing_party/party.py

```diff
@@ -1,23 +1,121 @@
 # ------------- WAVE 1 --------------------
 
+from tokenize import generate_tokens
+
+
 def create_movie(title, genre, rating):
-    pass
 
-# -----------------------------------------
+    if title == None or genre == None or rating == None:
+        return None
+
+    new_movie = {"title": title, "genre": genre, "rating": rating}
+
+    return new_movie
+
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+
+    return user_data
+
+def watch_movie(user_data, title):
+    for item in user_data["watchlist"]:
+        if item["title"] == title:
+            user_data["watchlist"].remove(item)
+            user_data["watched"].append(item)
+    return user_data
+
+
 # ------------- WAVE 2 --------------------
-# -----------------------------------------
 
+def get_watched_avg_rating(user_data):
+    ratings = 0
+
+    if user_data["watched"] == []:

```

Comment:
Nice guard clause here - `if not user_data["watched"]:` is a more pythonic way to check if the list is empty